Index: src/main/c++/udt/src/udt.h
===================================================================
--- src/main/c++/udt/src/udt.h	(revision 4015)
+++ src/main/c++/udt/src/udt.h	(working copy)
@@ -47,10 +47,11 @@
    #include <sys/socket.h>
    #include <netinet/in.h>
 #else
-   #include <windows.h>
    #ifdef __MINGW__
+      #include <stdint.h>
       #include <ws2tcpip.h>
    #endif
+   #include <windows.h>
 #endif
 #include <fstream>
 #include <set>
